samples/GoogleHangoutsChat.gs

```diff
@@ -0,0 +1,90 @@
+var PRIVATE_KEY =
+    '-----BEGIN PRIVATE KEY-----\n...\n-----END PRIVATE KEY-----\n';
+var CLIENT_EMAIL = '...';
+
+/**
+ * Authorizes and makes a request to the Hangouts Chat API
+ * for sending a text message to all DM room bot has been added.
```

Comment:
The phrase "all DM room" is confusing. I think "all DMs" is better, since "rooms" are something else entirely.